packages/react-core/src/components/Select/Select.tsx

```diff
@@ -143,6 +143,8 @@ export interface SelectProps
   maxHeight?: string | number;
   /** Icon element to render inside the select toggle */
   toggleIcon?: React.ReactElement;
+  /** Custom icon for the dropdown replacing the CaredDownIcon */
+  selectToggleIcon?: React.ReactElement;
```

Comment:
This is called `toggleIndicator` in the dropdown component. It would probably be good to align with that. https://github.com/patternfly/patternfly-react/blob/5538c2f9925a8ac29505f1a4ec6003add9e14fa0/packages/react-core/src/components/Dropdown/DropdownToggle.tsx#L38-L39